autogtp/main.cpp

```diff
@@ -196,20 +198,28 @@ int main(int argc, char *argv[])
 #else
     QTextStream cerr(stderr, QIODevice::WriteOnly);
 #endif
-
     cerr << "autogtp v" << AUTOGTP_VERSION << endl;
 
+    auto start = std::chrono::high_resolution_clock::now();
     auto success = true;
-    auto games_played = 0;
+    long games_played = 0;
```

Comment:
Please keep the coding style: auto games_played = long{0}; (or use a literal suffix)